### PR TITLE
Add `derivatives` extension for `futures` and `options`

### PR DESCRIPTION
**Pull request by the-praxs**
Adds a `derivatives` extension that works as a separate command for accessing the `options` and `futures` commands.

### CODE REVIEW

File: openbb_platform/extensions/derivatives/README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 ## Installation
 
 To install the extension, run the following command in this folder:
 
 ```bash
-pip install openbb-futures
+pip install openbb-derivatives
 ```
 
 For development please check [Contribution Guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/feature/openbb-sdk-v4/openbb_platform/CONTRIBUTING.md).
```
Comment (piiq) [inline openbb_platform/extensions/derivatives/README.md:13]: update url to point to develop, plz

File: openbb_platform/extensions/derivatives/integration/test_derivatives_python.py [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
+    assert isinstance(result, OBBject)
+    assert len(result.results) > 0
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        ({"symbol": "AAPL"}),
+        ({"provider": "intrinio", "source": "delayed", "symbol": "AAPL"}),
+        # ({"provider": "intrinio", "symbol": None, "source": "delayed"}),
```
Comment (piiq) [inline openbb_platform/extensions/derivatives/integration/test_derivatives_python.py:40]: remove commented out lines please

**Review by piiq — APPROVED** [review]
I left 2 minor comments not related to the code. Thank you

**Comment by the-praxs** [comment]
> I left 2 minor comments not related to the code. Thank you

Thanks for pointing them out!

**Pull request merged**